crates/iota-config/src/node.rs

```diff
@@ -1232,3 +1231,38 @@ impl RunWithRange {
         matches!(self, RunWithRange::Checkpoint(seq) if *seq == seq_num)
     }
 }
+
+/// A helper struct used with #[serde_as(as = ...)] to change the
+/// de/serialization format of an `IotaKeyPair` to Bech32 when written to or
+/// read from the node config.
+#[expect(dead_code)]
+struct IotaKeyPairConfig;
```

Comment:
I think I would prefer just using the standard [serde with attribute](https://serde.rs/field-attrs.html#with) and a small module with these fns in it

Reply:
Too bad, I actually like it better this way 🤷‍♂️ . Anyone else optioniated about that?

Reply:
If this version forces us to have a `#[expect(dead_code)]`, I'd go with the other one